packages/gasket-preset-api/package.json

```diff
@@ -27,7 +27,9 @@
     "@gasket/plugin-https": "^6.45.0",
     "@gasket/plugin-lint": "^6.45.0",
     "@gasket/plugin-log": "^6.45.0",
-    "@gasket/plugin-swagger": "^6.45.0"
+    "@gasket/plugin-logger": "^6.43.0",
+    "@gasket/plugin-swagger": "^6.45.0",
+    "@gasket/plugin-winston": "^6.43.0"
```

Comment:
So, we don't need both the old `log` and the new `logger`. But given `logger` will be a default plugin of the CLI, neither of these should be in the preset. Only `winston` makes sense to add.